### PR TITLE
Removing bool functions and styling.

### PR DESCRIPTION
**Pull request by mltejera**
Removing functions that take a boolean as a parameter. Instead passing in the state object.

Removing styles and their references.

### CODE REVIEW

File: pomodorotimer/src/components/SingleTimer.js [hunk 1]
```diff
@@ -119,16 +115,16 @@ export default class SingleTimer extends React.Component {
     }
 }
 
-function statusIcon(isRunning, isComplete) {
-    if (isRunning) {
-        return <CircularProgress className={Styles.progressSpinner} />
-    } else if (isComplete) {
-        return <DoneIcon className={Styles.doneIcon} />
+function statusIcon(state) {
```
Comment (dankreft) [inline pomodorotimer/src/components/SingleTimer.js]: Don't take in the whole thing...deconstruct:

```js
function statusIcon({ isRunning, isComplete }) {
```

File: pomodorotimer/src/components/SingleTimer.js [hunk 2]
```diff
@@ -119,16 +115,16 @@ export default class SingleTimer extends React.Component {
...
+    if (state.isRunning) {
+        return <CircularProgress />
+    } else if (state.isComplete) {
```
Comment (dankreft) [inline pomodorotimer/src/components/SingleTimer.js]: It doesn't make sense to have 'else if' if the preceding conditional exits the function.

```js
if ( isRunning ) {
  return ...
}

if ( isComplete ) {
 return ...
}
```

Comment (mltejera) [reply pomodorotimer/src/components/SingleTimer.js]: right. Coming up through the C world, I was coached to only have a single return statement....old habits.

File: pomodorotimer/src/components/SingleTimer.js (hunk 1 above)
Comment (mltejera) [reply pomodorotimer/src/components/SingleTimer.js]: That's a neat little language feature!

**Pull request merged**